service/history/workflow/update/registry.go

```diff
@@ -161,8 +155,6 @@ func WithTracerProvider(t trace.TracerProvider) Option {
 	}
 }
 
-var _ Registry = (*registry)(nil)
```

Comment:
`var`s are expected at the top of the file, I believe.

Reply:
Generally yes, but not for this type of check. I prefer it to be after type declaration.